packages/api-client/src/createBasicHeader.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+export function createBasicHeader(username: string, password: string): string {
+    // TODO: Replace with @tupaia/utils createBasicHeader() when we unpublish
+    return `Basic ${Buffer.from(`${username}:${password}`).toString('base64')}`;
```

Comment:
Just since I saw it, we've stopped publishing the `@tupaia/api-client` package, so I think this can just be switched to using `@tupaia/utils`